source/Nevermore/AST/Select.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Nevermore.AST
+{
+    public class Select : ISelect
+    {
+        readonly IRowSelection rowSelection;
+        readonly ISelectColumns columns;
+        readonly ISelectSource from;
+        readonly Where where;
+        readonly OrderBy orderBy; // Can be null
+
+        public Select(IRowSelection rowSelection, ISelectColumns columns, ISelectSource from, Where where, OrderBy orderBy)
+        {
+            this.rowSelection = rowSelection;
+            this.columns = columns;
+            this.from = from;
+            this.where = where;
+            this.orderBy = orderBy;
+        }
+
+        public string GenerateSql()
```

Comment:
It would be nice if `ToString` was overridden to point at `GenerateSql`, that way you can see the query easily in the debugger. `DebuggerView` could also be used to show the main properties of each type, again to help debugging when viewing the AST.

Reply:
Done. I only added `ToString` - this felt the most efficient summary of the properties on each AST node anyway, so I didn't think anything more was required in most cases.